inst/examples/app/analytics/R/dashboard-component.R

```diff
@@ -0,0 +1,197 @@
+# Dashboard header carrying the title of the dashboard
+
+# Header --------------------------------------------------------------------------------------
+
+header_telemetry <- semantic.dashboard::dashboardHeader(
+  shinyjs::useShinyjs(),
+  shiny::suppressDependencies("bootstrap"),
+  shiny::suppressDependencies("plotlyjs"),
+  style = "min-height: 100%;",
+  shiny::tags$head(
+    shiny::tags$link(rel = "stylesheet", href = "www/styles.css"),
+    shiny::tags$script(src = "https://cdn.plot.ly/plotly-1.20.2.min.js"),
+    shiny::tags$script(
+      src = "https://cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.4/numeral.min.js"
+    )
+  ),
+  shiny::tags$div(
+    class = "right menu",
+    shiny::tags$a(
+      class = "icon item", href = "../../logout",
+      semantic.dashboard::icon("power off")
+    )
+  )
+)
+
+
+# Sidebar -------------------------------------------------------------------------------------
+
+render_download_button <- function(output_id, label, style = NULL) {
+  shiny::a(
+    id = output_id,
+    class = "ui grey tiny basic button shiny-download-link",
+    style = style,
+    href = "",
+    target = "_blank",
+    download = NA,
+    label
+  )
+}
```

Comment:
I wanted to include this in `inst/examples/app/analytics/R/ui-component.R`, but the app could not find this function properly. Moving it here makes the app work better. This came from `R/stats-panel-ui.R`.